### PR TITLE
add task and resume about javascript refreshment and data structure

### PR DESCRIPTION
**Pull request by BagusAri-17**
Hello sir, please review my task and resume about javascript refreshment and data structure. Thankyou.

### CODE REVIEW

File: 07_javascript-refreshment/praktikum/createProduct.html [hunk 1]
```diff
@@ -0,0 +1,260 @@
...
+                        let category = document.querySelector("#product-category").value;
+                        let image = document.querySelector("#product-image").value;
+                        let freshnesss = document.querySelectorAll(
+                            'input[name="freshness"]'
+                        );
+                        let selectFreshness = "";
+                        let description = document.querySelector(
+                            "#product-description"
+                        ).value;
+                        let price = document.querySelector("#product-price").value;
```
Comment (syukronarie) [inline 07_javascript-refreshment/praktikum/createProduct.html:196]: jika variable tidak direassign gunakan const

File: 08_introduction-data-structure/README.md
Comment (syukronarie) [inline 08_introduction-data-structure/README.md:1]: 👍

**Review by syukronarie — APPROVED** [review]

**Comment by syukronarie** [comment]
pelajari event submit dan change event

File: 07_javascript-refreshment/praktikum/createProduct.html (hunk 1 above)
Comment (BagusAri-17) [reply 07_javascript-refreshment/praktikum/createProduct.html:196]: Baik mas, terima kasih atas reviewnya.

**Pull request merged**